opensrp-app/src/main/java/org/smartregister/SyncConfiguration.java

```diff
@@ -186,4 +188,6 @@ public int getMaxAuthenticationRetries() {
     public boolean firebasePerformanceMonitoringEnabled() {
         return false;
     }
+
+    public List <Pair<String,String>> getGlobalSettingsQueryParams() {return null;}
```

Comment:
Please add java docs on what this configures.

Reply:
updated